src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -40,8 +40,8 @@ public void openAndCloseNavigationAndItsSubmenu() {
     topBar.closeNavigation();
   }
 
-  @Test(groups = "navigation-validSearchPhraseTriggersSuggestionThatCanBeOpened")
-  public void validSearchPhraseTriggersSuggestionThatCanBeOpened() {
+  @Test(groups = "mercury_navigation_validSearchPhraseTriggersSuggestionThatCanBeOpened")
+  public void mercury_navigation_validSearchPhraseTriggersSuggestionThatCanBeOpened() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)